lib/Pulse.Lib.AVLTree.fst

```diff
@@ -0,0 +1,948 @@
+//Pulse AVL tree implementation. Inspired from Steel. The FStar spec file is adopted from Steel
+//----------------------------------------------------------------------------------------------------------
+module Pulse.Lib.AVLTree
+open Pulse.Lib.Pervasives
+module S = Pulse.Lib.Stick.Util
```

Comment:
This import and the next one are not used in the module as far as I can see.

Reply:
comment addressed.